lib/req-res/status.js

```diff
@@ -4,59 +4,84 @@ module.exports = function ( options ) {
 
 	var respond = options.respond;
 	var log     = options.log;
+	var emitter = options.emitter;
 
-	return {
-		'success' : function ( data ) {
+	var responded = false;
 
-			respond( {
-				'status' : 'success',
-				'data'   : data
+	var timeout = setTimeout( function () {
+		responded = true;
+		respond( {
+			'status' : 'fail',
+			'data'   : 'Request timeout'
+		} );
+		setImmediate( function () {
+			log.silly( 'success', {
```

Comment:
@openam addressed it here [54](https://github.com/School-Improvement-Network/lapin/pull/54)